### PR TITLE
monitor,server: add segment download time metric

### PR DESCRIPTION
**Pull request by kyriediculous**
**What does this pull request do? Explain your changes. (required)**
Adds download time and segments downloaded as a metric 

**Checklist:**
- [ ] README and other documentation updated
- [x] Node runs in OSX and devenv
- [x] All tests in `./test.sh` pass

### CODE REVIEW

File: monitor/census.go [hunk 1]
```diff
@@ -453,6 +464,12 @@ func InitCensus(nodeType, nodeID, version string) {
 			TagKeys:     baseTags,
 			Aggregation: view.Distribution(0, .10, .20, .50, .100, .150, .200, .500, .1000, .5000, 10.000),
 		},
+		{
+			Name:        "download_time_seconds",
+			Measure:     census.mDownloadTime,
```
Comment (darkdarkdragon) [inline monitor/census.go:469]:
```suggestion
			Measure:     census.mDownloadTime,
			Description: "Download (from orchestrator) time",
```
I think this description is used in /metrics output

**Review by darkdarkdragon — CHANGES_REQUESTED** [review]
One comment, otherwise looks good

File: monitor/census.go (hunk 1 above)
Comment (kyriediculous) [reply monitor/census.go:469]: fixed in 604fab7

**Review by darkdarkdragon — APPROVED** [review]
LGTM

**Pull request merged**